lib/utils/ember.js

```diff
@@ -298,16 +298,55 @@ function isObserverProp(node) {
   return isPropOfType(node, 'observer');
 }
 
-function isComputedProp(node) {
+function isComputedProp(
+  node,
+  importedEmberName,
+  importedComputedName,
+  { includeSuffix, includeMacro } = {}
+) {
   const allowedMemberExpNames = new Set(['volatile', 'readOnly', 'property', 'meta']);
-  if (types.isMemberExpression(node.callee) && types.isCallExpression(node.callee.object)) {
-    return (
-      isModule(node.callee.object, 'computed') &&
+  return (
+    // computed
+    (types.isIdentifier(node) && node.name === importedComputedName) ||
+    // computed()
+    (types.isCallExpression(node) &&
+      types.isIdentifier(node.callee) &&
+      node.callee.name === importedComputedName) ||
+    // Ember.computed()
+    (types.isCallExpression(node) &&
+      types.isMemberExpression(node.callee) &&
+      types.isIdentifier(node.callee.object) &&
+      node.callee.object.name === importedEmberName &&
       types.isIdentifier(node.callee.property) &&
-      allowedMemberExpNames.has(node.callee.property.name)
-    );
-  }
-  return isModule(node, 'computed');
+      node.callee.property.name === 'computed') ||
+    // Ember.computed().readOnly() or computed().readOnly()
```

Comment:
These should be `computed.` rather than `computed().`

Reply:
Oh actually I will change this comment to say:

```
// Ember.computed().volatile() or computed().volatile()
```

to avoid confusion with the macro.

Reply:
Ah. It could also help to do `Ember.computed(...).volatile()`, since it isn't valid without a function as an argument to the `computed` call.